configs/config.opusmt-multilingual-test.yml

```diff
@@ -0,0 +1,88 @@
+####
+# Example of a production config
+# Change language pair, experiment name, datasets and other settings if needed
+# Training low resource languages might require more tuning of pipeline/training/configs
+###
+
+
+experiment:
+  name: opusmt
+  src: en
+  trg: sw
+  src_three_letter: eng
+  trg_three_letter: swa
```

Comment:
Putting this here instead of calling `mtdata` every time we need it is a nice improvement!